DRAFT_RELEASE_NOTES.md

```diff
@@ -78,5 +79,8 @@ To start your aiSSEMBLE upgrade, update your project's pom.xml to use the 1.11.0
     - **NOTE:** This will update any aiSSEMBLE dependencies in 'pyproject.toml' files automatically
 3. Repeat the previous step until all manual actions are resolved
 
+#### Maven Docker Build
```

Comment:
Q/S: Why wouldn't this be under **_Precondition Steps_**? Historically this finalization section has been very light so I'm not sure many projects will be trained to pay attention to this.

Reply:
Yeah.. I can move it up coz original thought was this is not a required step